examples/rapids/mlflow_project/README.md

```diff
@@ -38,7 +38,7 @@ MLflow to log models and deploy them as local REST API endpoints for real-time i
                --experiment-name RAPIDS-CLI \
                -P max_depth=10 -P max_features=0.75 -P n_estimators=500 \
                -P conda-env=$PWD/envs/conda.yaml \
-               -P fpath=airline_small.parquet
+               -P fpath=iris.csv
```

Comment:
The iris dataset might not be the best dataset to demonstrate the power of `cuDF`.